index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
+        <nav>
+          <ul class="Menu-bar">
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+          </ul>
```

Comment:
- [ ] I suggest that for better practice we consider the use of icons and hamburger image ☺️ . However, most importantly, you should ensure that you try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=tFTlxlBLvFLIXvcT-0). Consider aligning the hamburger to the right as seen in the Figma design.

Your design 😎 
![humb](https://user-images.githubusercontent.com/99394374/218770662-02b75ccb-5eab-48f3-a3c4-dfe80ca0bcff.png)


Figma design 😎 
![humb fig](https://user-images.githubusercontent.com/99394374/218770740-2b040d54-1950-46f8-95c9-1b06c3537554.png)